NitroxLauncher/OptionPage.xaml

```diff
@@ -4,40 +4,46 @@
       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
       xmlns:local="clr-namespace:NitroxLauncher"
-      mc:Ignorable="d" 
-      d:DesignHeight="470" d:DesignWidth="719"
+      mc:Ignorable="d" d:DesignHeight="470" d:DesignWidth="719"
```

Comment:
Is the autoformatter putting everything on the same line?

Reply:
@iCleeem This is a legitimate question, not a "please revert this".

Reply:
@MarijnS95 No the autoformatter did not put everything on the same line, I decided to modify this on my own